Source/WebInspectorUI/UserInterface/Protocol/Target.js

```diff
@@ -140,7 +140,6 @@ WI.Target = class Target extends WI.Object
     get DOMAgent() { return this._agents.DOM; }
     get DOMDebuggerAgent() { return this._agents.DOMDebugger; }
     get DOMStorageAgent() { return this._agents.DOMStorage; }
-    get DatabaseAgent() { return this._agents.Database; }
```

Comment:
im slightly hesitant about removing this from the UI as it means we'll be dropping support when the backend technically could still work, but seeing as how this all has been disabled for so long i think it's fine

if we are gonna remove the frontend bits, however, we should manually edit all of the `Source/WebInspectorUI/UserInterface/Protocol/Legacy/*/*/InspectorBackendCommands.js` to not have the `Database` domain as otherwise we'll still create the `WI.DatabaseObserver` class, `DatabaseAgent` class, etc. which is just wasteful